### PR TITLE
ERMS-2468, ERMS-2463

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/i18n/messages.properties [hunk 1]
```diff
@@ -1820,7 +1823,7 @@ subscription.packages.config.label = Setting for package <strong>{0}</strong>
 subscription.packages.changeType.label = Change type
 subscription.packages.setting.label = Setting
 subscription.packages.notification.label = Notification
-subscription.packages.auditable = Inheritance active
+#subscription.packages.auditable.label = Inherit
```
Comment (croqueGrec09) [inline app/grails-app/i18n/messages.properties:1826]: The translation string is in use

File: app/grails-app/i18n/messages.properties [hunk 2]
```diff
@@ -2173,41 +2176,34 @@ subscription.properties.public = Public properties
...
-subscription.copyLinks = Link copy with same organisations
-subscription.copyLicense = Link copy with same license
-subscription.copylinktoSubscription= Link copy with same parent subscription
-subscription.copyPackages = Link copy with same packages
-subscription.copyPackageSettings = Apply same package settings
+subscription.copyLinks = Kopie mit denselben Organisationen verknüpfen
+subscription.copyLicense = Kopie mit demselben Vertrag verknüpfen
+subscription.copylinktoSubscription= Kopie mit derselben Elternlizenz verknüpfen
+subscription.copyPackages = Kopie mit den gleichen Paketen verknüpfen
 subscription.copyIds = Copy identifiers
```
Comment (croqueGrec09) [inline app/grails-app/i18n/messages.properties:2186]: Overwrite English strings by German translations in English file??!

File: app/grails-app/domain/com/k_int/kbplus/UserSettings.groovy [hunk 3]
```diff
@@ -48,7 +48,8 @@ class UserSettings {
         IS_REMIND_FOR_ORG_PRIVATE_PROP              (RefdataValue, RDConstants.Y_N),
         IS_REMIND_FOR_PERSON_PRIVATE_PROP           (RefdataValue, RDConstants.Y_N),
         IS_REMIND_FOR_TASKS                         (RefdataValue, RDConstants.Y_N),
-        IS_REMIND_FOR_SURVEYS_ENDDATE               (RefdataValue, RDConstants.Y_N),
+        IS_REMIND_FOR_SURVEYS_NOT_MANDATORY_ENDDATE (RefdataValue, RDConstants.Y_N),
```
Comment (croqueGrec09) [inline app/grails-app/domain/com/k_int/kbplus/UserSettings.groovy:51]: Domain changes in RC ... is this really hotfix-commanded?

Comment (Moetez16) [reply app/grails-app/domain/com/k_int/kbplus/UserSettings.groovy:51]: YES

**Pull request merged**